care/facility/migrations/0337_auto_20230327_1319.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.11 on 2023-03-27 07:49
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('facility', '0336_auto_20230222_1602'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='assetlocation',
+            unique_together={('name', 'facility')},
```

Comment:
We cant merge this without checking if there are any current cases like this in production, in which case the migration would fail, @mathew-alex this needs to be checked manually to ensure we don't break anything

Reply:
@vigneshhari Wouldn't a Validation do the trick instead of having a Constraint?

Reply:
Just validation should do the trick.

Reply:
Switched to just validation.